### PR TITLE
not satisfied with being simply humongous, I've made the logo tremendous

### PR DESCRIPTION
**Pull request by michaelficarra**
I, personally, think the size of the logo should be directly related to the greatness of the language. If anything, we should strive to make it larger over time. The fact that this logo is of exceptional size reflects on how exceptional the language is.

### CODE REVIEW

**Comment by tkellen** [comment]
🤣

File: spec.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
   margin: 0; padding: 0;
 }
 #ecma-logo {
-  width: 500px;
+  width: 110%;
```
Comment (ljharb) [inline spec.html]: needs more zeroes

Comment (bterlson) [inline spec.html]: I mean, it is an SVG. We should take advantage of that.

Comment (ljharb) [inline spec.html]: 1e262%

Comment (michaelficarra) [inline spec.html]: That's absurd. In a survey of other standards documents, none of them use more than 4 characters to describe the logo width. :upside_down_face:

**Comment by michaelficarra** [comment]
I, personally, think the size of the logo should be directly related to the greatness of the language. If anything, we should strive to make it larger over time. The fact that this logo is of exceptional size reflects on how exceptional the language is.

**Comment by claudepache** [comment]
Exceptional, indeed. Rare are the languages who have two distinct values for nil, or whose type-checking operator assimilates  half of nils to objects.

**Pull request closed without merging**